lucid1/src/Lucid/Base.hs

```diff
@@ -81,7 +80,7 @@ import qualified Data.Set as Set
 -- 'makeAttribute'.  Attributes are case sensitive, so if you want
 -- attributes to be merged properly, use a single case representation.
 data Attribute = Attribute !Text !Text
-  deriving (Show,Eq,Typeable)
```

Comment:
Fixes [`-Wderiving-typeable`](https://downloads.haskell.org/ghc/9.12.1-alpha1/docs/users_guide/using-warnings.html#ghc-flag-Wderiving-typeable) which exists since 7.10